csv/src/fs2/data/csv/package.scala

```diff
@@ -16,8 +16,9 @@
 package fs2
 package data
 
-import csv.internals._
+import java.nio.{CharBuffer => JCharBuffer}
 
+import csv.internals._
```

Comment:
I like having my personal relative imports at the beginning and the Java ones at the end (yes, I'm that kind of guy :)).